backend/src/handlers_prelude/github_list_branches.rs

```diff
@@ -0,0 +1,37 @@
+//! Getting Github Branches
+use axum::{
+    extract::State,
+    http::StatusCode,
+    Json, Router,
+};
+use axum::routing::get;
+use tracing::{error, info};
+use serde_json::json;
+use crate::gh::list_branches;
+use crate::AppState;
+use color_eyre::Result;
+
+pub async fn list_branches_handler(
+    State(state): State<AppState>,
+) -> Result<(StatusCode, Json<serde_json::Value>), (StatusCode, Json<serde_json::Value>)> {
```

Comment:
Same as the other one, a more strongly typed json plz

Reply:
Can you explain more what you mean by a more strongly typed json?

Reply:
Just using a struct that implements serialize/de serialize, rather than a `serde_json::Value`. Again, it's fine if you would prefer to just switch it to a `serde_json::Object`, but at that point I'd ask that you explain what exactly it returns in the function docstring.